flocker/control/httpapi.py

```diff
@@ -510,19 +549,22 @@ def create_container_configuration(
 
         # Check if container by this name already exists, if it does
         # return error.
-
         for node in deployment.nodes:
             for application in node.applications:
                 if application.name == name:
                     raise CONTAINER_NAME_COLLISION
 
+        # Find the volume, if any:
+        attached_volume = None
+        if volumes:
+            attached_volume = self._get_attached_volume(host, volumes[0])
```

Comment:
Might be worth adding a comment just to underline again that we only support one attached volume right now, hence why we only care about `volumes[0]`